### PR TITLE
(colab) Add 'list' and 'delete' to tbdev

### PR DESCRIPTION
**Pull request by bileschi**
Extends getting started example to include 'list' and 'delete' subcommands.

* Motivation for features / changes
Users didn't know they could list / delete.

* Technical description of changes
Adds a few new cells.

* Screenshots of UI changes
image

### CODE REVIEW

File: docs/tbdev_getting_started.ipynb [hunk 1]
```diff
@@ -1,36 +1,25 @@
 {
```
Comment (stephanwlee) [inline docs/tbdev_getting_started.ipynb]: <p>The change seems identical except for the newline. Revert the change?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/2992/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (stephanwlee) [inline docs/tbdev_getting_started.ipynb]: <p>Put <code>tensorboard dev delete --experiment_id pQpJNh00RG2Lf1zOe9BrQA </code> around pretty quotes for code?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/2992/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

File: docs/tbdev_getting_started.ipynb [hunk 2]
```diff
@@ -187,16 +174,84 @@
...
+        "colab_type": "code",
+        "id": "VSkJTT9rNWJq"
+      },
+      "outputs": [],
+      "source": [
+        "# You must replace YOUR_EXPERIMENT_ID with the value output from the previous\n",
+        "# tensorboard `list` command or `upload` command.  For example\n",
+        "# tensorboard dev delete --experiment_id pQpJNh00RG2Lf1zOe9BrQA\n",
+        "\n",
+        "!tensorboard dev delete --experiment_id YOUR_EXPERIMENT_ID_HERE"
```
Comment (stephanwlee) [inline docs/tbdev_getting_started.ipynb]: Btw, this may cause the doc compilation to fail since it will exit with non-zero.

File: docs/tbdev_getting_started.ipynb (hunk 1 above)
Comment (bileschi) [reply docs/tbdev_getting_started.ipynb]: This the the format as output by Cider.  I'm reluctant to go through and manually override.

Comment (bileschi) [reply docs/tbdev_getting_started.ipynb]: <p>Is there a way to use ReviewNB to selectively revert diff blocks?  Can ReviewNB author commits?</p>

Comment (stephanwlee) [reply docs/tbdev_getting_started.ipynb]: > This the the format as output by Cider. I'm reluctant to go through and manually override.

I think the kosher way to edit these is to use Jupyter or Colab to for the right serialization and right output cell cache content.
